ui/partials/perf/testdatachooser.html

```diff
@@ -3,19 +3,46 @@
   <h3 class="modal-title">Add test data</h3>
 </div>
 <div class="modal-body">
-  <h4>Project</h4>
-  <select class="form-control" ng-change="updateTestInput()" ng-model="selectedProject" ng-options="project.name for project in projects">
-  </select>
-  <h4>Platform</h4>
-  <p class="blink" ng-show="loadingTestData">Loading series data...</p>
-  <select ng-hide="loadingTestData" ng-change="updateTestSelector()" ng-model="selectedPlatform" ng-options="platform for platform in platformList">
-  </select>
-  <h4>Test</h4>
-  <p class="blink" ng-show="loadingTestData">Loading series data...</p>
-  <div id="scrollable-dropdown-menu">
-    <input my-input ng-hide="loadingTestData" class="typeahead" type="text" placeholder="Enter test keywords (e.g. 'tp5o')"></input>
+  <div class="test-chooser">
+    <h4>Project</h4>
+    <select class="form-control" ng-change="updateTestInput()" ng-model="selectedProject" ng-options="project.name for project in projects">
+    </select>
+    <h4>Platform</h4>
+    <p class="blink" ng-show="loadingTestData">Loading series data...</p>
+    <select ng-hide="loadingTestData" ng-change="updateTestSelector()" ng-model="selectedPlatform" ng-options="platform for platform in platformList">
+    </select>
+    <h4>Tests</h4>
+    <div class="form-group">
+        <input class="form-control input-sm" type="text" ng-model="test_filter" placeholder="Filter tests"/>
+    </div>
+    <div id="choose-test-list" class="form-group">
+      <select multiple class="form-control"  ng-model="testList">
+         <option value="{{::left_elem}}" ng-repeat="left_elem in LeftList| filter: test_filter track by $id(left_elem)">
```

Comment:
Can you remove all the left/right stuff from this file?

In this case for example, we might want to do something like:

    <option value="{{::testElem}}" ng-repeat="testElem in testList| filter: testFilter track by $id(testElem)">

Please follow that same convention elsewhere. left/right is not very meaningful to people trying to understand what's going on here. :)